bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -146,32 +149,39 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 }
                 break;
             case RenaultBindingConstants.CHANNEL_HVAC_STATUS:
-                // We can only trigger pre-conditioning of the car.
-                if (command instanceof StringType && command.toString().equals(Car.HVAC_STATUS_ON)
-                        && !car.isDisableHvac()) {
-                    final MyRenaultHttpSession httpSession = new MyRenaultHttpSession(this.config, httpClient);
-                    try {
-                        updateState(CHANNEL_HVAC_STATUS, new StringType(Car.HVAC_STATUS_PENDING));
-                        car.resetHVACStatus();
-                        httpSession.initSesssion(car);
-                        httpSession.actionHvacOn(car.getHvacTargetTemperature());
-                        if (pollingJob != null) {
-                            pollingJob.cancel(true);
+                if (!car.isDisableHvac()) {
+                    if (command instanceof RefreshType) {
```

Comment:
Can you check lines 132-134 also? I know you didn't change them, but since you are implementing refresh, you probably want it to work for **CHANNEL_HVAC_TARGET_TEMPERATURE** also, and the implementation is wrong here.

Reply:
The CHANNEL_HVAC_TARGET_TEMPERATURE is a bit strange.  It only sets this value in the binding "car" object where is is persisted.  The value is not sent to the real car.  This value is only used on the HVAC ON preconditioning action. So there should be not polling job change when this is refreshed or set.

Reply:
Oh. Now I checked it more thoroughly and you are right. I'm also right though, implementation is wrong or a least very odd. But that's outside the scope of your PR to fix:
- If the value cannot be requested from the car, there is no reason for implementing refresh in `handleCommand`. There is nothing to refresh.
- If HVAC is already ON, setting a different target temperature should call `actionHvacOn` to update the value in the car.

I would suggest to leave this for now (in context of this PR), but it seems there is room for improvement here.

Reply:
As a result of testing the target temperature has to be 19, 20 or 21... or HVAC ON it does not consistently work.  So it is not work having as a channel.  However maybe one day it will be more useful.  I am not sure what will happen if you try to change the target temperature once the car is doing a pre-heat.  I would need to test this.

Reply:
Let's leave this for this PR, you could then improve this further in a new PR if you would like to.